Makefile

```diff
@@ -363,6 +364,13 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 undeploy: kustomize ## Undeploy controller from the K8s cluster specified in ~/.kube/config. Call with ignore-not-found=true to ignore resource not found errors during deletion.
 	$(KUSTOMIZE) build config/profile/$(PROFILE) | $(KUBECTL) delete --ignore-not-found=$(ignore-not-found) -f -
 
+.PHONY: plugin-infra
+plugin-infra:
```

Comment:
Would it make sense to have a target for uninstalling the plugin infra from the cluster?

Reply:
I do not consider it as very important :) but added.